specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Dataset.json

```diff
@@ -593,8 +597,7 @@
         }
       },
       "required": [
-        "location",
-        "sheetName"
+        "location"
```

Comment:
Is this change intentional? Can't quite deduce from the title of the change. There is no description with the PR so I am just guessing at this point but this is a behavior change.

Reply:
This is synchronizing the property between DataFactory and Synapse. We should avoid using two different set of definitions for the same property.